src/gui/Layout.cpp

```diff
@@ -293,25 +293,24 @@ auto Layout::getPaddingLeftOfPage(size_t pageIndex) const -> int {
     bool isPairedPages = this->mapper.isPairedPages();
     const Settings* settings = this->view->getControl()->getSettings();
 
-    auto const paddingBefore =
-            sumIf(XOURNAL_PADDING, settings->getAddVerticalSpaceAmount(), settings->getAddHorizontalSpace());
+    auto paddingBefore =
+            sumIf(XOURNAL_PADDING, settings->getAddHorizontalSpaceAmount(), settings->getAddHorizontalSpace());
 
     auto const pageXLocation = as_signed(this->mapper.at(pageIndex).first);
 
     // No page pairing or we haven't rendered enough pages in the row for
     // page pairing to have an effect,
-    if (!isPairedPages || pageXLocation == 0) {
-        return strict_cast<int>(pageXLocation * XOURNAL_PADDING_BETWEEN + as_signed(paddingBefore));
+    if (!isPairedPages) {
+        return strict_cast<int>(pageXLocation * XOURNAL_PADDING_BETWEEN + XOURNAL_PADDING_BETWEEN / 2 +
+                                as_signed(paddingBefore));
```

Comment:
Page 0 should have no `XOURNAL_PADDING_BETWEEN` as offset from the left view anchor.
Where comes the `XOURNAL_PADDING_BETWEEN / 2` from?

Reply:
Also the `XOURNAL_PADDING_BETWEEN` is the distance between two pages.

Reply:
My version is correct AFAICT. See the layout code, to understand how this actually plays out.